java/readme.md

```diff
@@ -0,0 +1,213 @@
+# Java code example
+
+
+## Prepare the client environment
+
+Download and install the JDK from [openjdk.org](https://openjdk.org/projects/jdk/21/) and Maven from [maven.apache.org](https://maven.apache.org/download.cgi). This example uses the [Apache Kafka](https://github.com/apache/kafka) client library.
```

Comment:
Nit: OpenJDK is just one of the options. Any JDK can be used. 

Also: what is the minimal version of the Apache Kafka client? I expect that many java users will simply use the Kafka client dependency instead of copy-pasting the whole pom.